component/src/main/java/com/celements/structEditor/StructuredDataEditorScriptService.java

```diff
@@ -153,4 +155,15 @@ public String getCellValueAsString(DocumentReference cellDocRef) {
     return propClass.transform(PROPCLASS_TO_API);
   }
 
+  public boolean isSelectMultiselect(DocumentReference cellDocRef) {
+    boolean isMultiselect = false;
+    try {
+      XWikiDocument cellDoc = modelAccess.getDocument(cellDocRef);
+      isMultiselect = modelAccess.getFieldValue(cellDoc, FIELD_IS_MULTISELECT).or(false);
```

Comment:
I think there is a version of getFieldValue where you can use the DocumentReference directly, without getting the XWikiDocument first